### PR TITLE
Support flashing of different chips locally

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -90,10 +90,20 @@ $(JAG_TOIT_DEPENDENCIES): $(SDK_BUILD_MARKER)
 # The SDK build marker is *not* phony, so we only
 # use the rule once per invocation of this Makefile.
 $(SDK_BUILD_MARKER):
-	make -C $(JAG_TOIT_REPO_PATH) version-file esp32 esptool
+	make -C $(JAG_TOIT_REPO_PATH) version-file esptool
+	make -C $(JAG_TOIT_REPO_PATH) esp32
+	make -C $(JAG_TOIT_REPO_PATH) ESP32_CHIP=esp32s2 esp32
+	make -C $(JAG_TOIT_REPO_PATH) ESP32_CHIP=esp32s3 esp32
+	make -C $(JAG_TOIT_REPO_PATH) ESP32_CHIP=esp32c3 esp32
```
Comment (kasperl) [inline Makefile:97]: I'd drop these changes and go back to `make -C ... version-file esp32 esptool`.

Comment (floitsch) [reply Makefile]: done.

File: Makefile [hunk 2]
```diff
@@ -94,6 +94,12 @@ $(SDK_BUILD_MARKER):
 	mkdir -p $(BUILD_DIR)
 	echo "$(BUILD_DATE)" > $@
 
+.PHONY: all-chips
+all-chips:
```
Comment (kasperl) [inline Makefile:98]: Should this depend on SDK_BUILD_MARKER?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: Makefile (hunk 2 above)
Comment (floitsch) [reply Makefile:98]: I think it could.
I already committed without it.
If we feel that it could improve performance we can change it.

**Pull request merged**